keps/sig-scheduling/3633-matchlabelselectors-to-podaffinity/README.md

```diff
@@ -717,7 +726,8 @@ In terms of Stable versions, users can choose to opt-out by not setting the
 
 ###### What happens if we reenable the feature if it was previously rolled back?
 
-Scheduling of new Pods is affected. (Scheduled Pods aren't affected.)
+Scheduling of new Pods created with `MatchLabelSelectors` is affected. 
```

Comment:
nit: Scheduling of newly created pods with `MatchLabelSelector` set is affected.
All already existing pods are unafected.